oxidation/crates/platform-async/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "platform-async"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+flume = "0.10.12"
+futures = "0.3.21"
+futures-lite = { version = "1.12.0", default-features = false }
+
+[dev-dependencies]
+log = "0.4.17"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
so far we use feature `web` vs `native` instead of relying on target_arch (given `wasm32` may refer to wasi, or to a node module)

I guess the advantage of the target_arch, though less consistent, is to avoid having to deal with feature and default_features in the crates but I have the feeling this is not a big deal given we have to set the crates once and for all with default_features=false for the dependencies and we're good ;-)

Reply:
I can use `target_family = "wasm"`. 

I prefer `target_*` over `feature = web / native` because it's more explicite that the architecture/capabilities will be different